cedar-policy/src/api.rs

```diff
@@ -1361,7 +1362,77 @@ pub struct SchemaFragment {
     lossless: cedar_policy_validator::json_schema::Fragment<cedar_policy_validator::RawName>,
 }
 
+fn annotations_to_pairs(annotations: &est::Annotations) -> impl Iterator<Item = (&str, &str)> {
+    annotations
+        .0
+        .iter()
+        .map(|(key, value)| (key.as_ref(), value.as_ref().map_or("", |a| a.as_ref())))
+}
+
 impl SchemaFragment {
+    /// Get annotations of a non-empty namespace.
+    /// We do not allow namespace-level annotations on the empty namespace
+    /// Returns `None` if `namespace` is not found in the [`SchemaFragment`]
```

Comment:
for better Markdown rendering
```suggestion
    /// Get annotations of a non-empty namespace.
    ///
    /// We do not allow namespace-level annotations on the empty namespace.
    ///
    /// Returns `None` if `namespace` is not found in the [`SchemaFragment`]
```